sys-auth/sssd/sssd-1.12.1.ebuild

```diff
@@ -136,5 +136,6 @@ src_test() {
 pkg_postinst(){
 	elog "You must set up sssd.conf (default installed into /etc/sssd)"
 	elog "and (optionally) configuration in /etc/pam.d in order to use SSSD"
-	elog "features. Please see howto in	http://fedorahosted.org/sssd/wiki/HOWTO_Configure_1_0_2"
+	elog "features. Please see howto in:"
+	elog "https://wiki.gentoo.org/wiki/Centralized_authentication_using_OpenLDAP#Client_PAM_configuration_SSSD_Method"
```

Comment:
A general hint for nice documentation should not go to the log. I would suggest to remove these lines.